src/cephadm/cephadm

```diff
@@ -239,6 +241,56 @@ logging_config = {
 }
 
 
+class ExcludeErrorsFilter(logging.Filter):
+    def filter(self, record: logging.LogRecord) -> bool:
+        """Only lets through log messages with log level below ERROR ."""
+        return record.levelno < logging.ERROR
+
+
+# During bootstrap operation we use:
+# stdout: for debug, info, etc
+# stderr: for errors
```

Comment:
do we want warnings to also be to stderr? Not saying we should or shouldn't, it's an open question imo. What do you think? I know the tracker only had error level messages in there but in a case like

```
[root@vm-00 ~]# ./cephadm --image quay.io/adk3798/ceph:testing bootstrap --mon-ip 4 1>out.txt 2>err.txt
[root@vm-00 ~]# cat out.txt 
Verifying podman|docker is present...
Verifying lvm2 is present...
Verifying time synchronization is in place...
Unit chronyd.service is enabled and running
Repeating the final host check...
podman (/usr/bin/podman) version 3.3.1 is present
systemctl is present
lvcreate is present
Unit chronyd.service is enabled and running
Host looks OK
Cluster fsid: 681de73c-b6d4-11ec-9f63-52540082b8b1
Address: 4 is not a valid IP address
Verifying IP 4 port 3300 ...
Address: 4 is not a valid IP address
[root@vm-00 ~]# cat err.txt 
ERROR: [Errno 99] Cannot assign requested address
```
would it be better to have those messages about not being a valid IP along with the error message rather than with the rest of the output since they're relevant to it (those messages are logged at warning level) or do we want to be really strict about what goes to stderr?

Reply:
@adk3798 that's a very good point. I wasn't either clear about it but after reading some more common practices I'd say warnings also have to go to stderr because:

1. The user can have all the information that cause (or lead to) errors in the stderr (and a lot of warnings do)
2. It's easier for the user to `redirect` everything which not a normal output to `stderr`
3. The user has not to perform speciall parsing if he wants just to pipe normal std output to another process such as `cephadm | my_other_process` . If we have warnings mixed with stderr that would be impossible without needing to filter out warnings (wich is not possible in this case since we don't have any keyword to use for filtering).

I'll update the core accordingly.